ci/cases/gfsv17/ocnanal.yaml

```diff
@@ -14,9 +14,10 @@ base:
   DO_VERFRAD: "YES"
   DO_VRFY_OCEANDA: "NO"
   FHMAX_GFS: 240
+  ACCOUNT: {{ 'HPC_ACCOUNT' | getenv }}
 
 ocnanal:
-  SOCA_INPUT_FIX_DIR: /scratch2/NCEPDEV/ocean/Guillaume.Vernieres/data/static/1440x1080x75/soca
+  SOCA_INPUT_FIX_DIR: /scratch1/NCEPDEV/global/glopara/fix/gdas/soca/20240624/1440x1080x75/soca
```

Comment:
There should be a machine agnostic way to do this, probably something like `{{ FIXgfs }}/gdas/soca/1440x1080x75/soca`

Reply:
Ha! Thanks for the pointer!

Reply:
This may require a change/addition to `sorc/link_workflow.sh` to make it happen though

Reply:
using your FIXgfs suggestion, thank @CoryMartin-NOAA